packages/BetterPhpDocParser/PhpDocInfo/PhpDocInfo.php

```diff
@@ -476,6 +476,9 @@ public function getNode(): \PhpParser\Node
         return $this->node;
     }
 
+    /**
+     * @param \PhpDocTagValueNode|null $phpDocTagValueNode
```

Comment:
this is invalid, there is no `\PhpDocTagValueNode` class, only `PHPStan\PhpDocParser\Ast\PhpDoc\PhpDocTagValueNode`, also, typed param already cover it.